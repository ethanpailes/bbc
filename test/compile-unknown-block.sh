#!/bin/sh

BBCC=$(stack exec which bbc)

${BBCC} -o unknown-block unknown-block.bb

exit $?
