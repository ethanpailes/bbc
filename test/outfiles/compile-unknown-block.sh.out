bbc: Unsupported langauge feature: Named types.
